packages/mobile/src/escrow/EscrowedPaymentReminderSummaryNotification.tsx

```diff
@@ -1,64 +1,46 @@
-import BaseNotification from '@celo/react-components/components/BaseNotification'
 import variables from '@celo/react-components/styles/variables'
 import * as React from 'react'
 import { WithNamespaces, withNamespaces } from 'react-i18next'
-import { Image, StyleSheet, View } from 'react-native'
+import { Image, StyleSheet } from 'react-native'
 import CeloAnalytics from 'src/analytics/CeloAnalytics'
 import { CustomEventNames } from 'src/analytics/constants'
 import { EscrowedPayment } from 'src/escrow/actions'
 import EscrowedPaymentLineItem from 'src/escrow/EscrowedPaymentLineItem'
+import { listItemRenderer } from 'src/escrow/EscrowedPaymentListScreen'
 import { Namespaces } from 'src/i18n'
 import { inviteFriendsIcon } from 'src/images/Images'
 import { navigate } from 'src/navigator/NavigationService'
 import { Stacks } from 'src/navigator/Screens'
+import SummaryNotification from 'src/notifications/SummaryNotification'
 
 interface OwnProps {
   payments: EscrowedPayment[]
 }
 
 type Props = OwnProps & WithNamespaces
 
-const PREVIEW_SIZE = 2
-
 export class EscrowedPaymentReminderSummaryNotification extends React.Component<Props> {
-  getTotal() {
-    return this.props.payments.length
+  onReview = () => {
+    CeloAnalytics.track(CustomEventNames.escrowed_payment_review)
+    navigate(Stacks.EscrowStack)
   }
 
-  getTitle() {
-    const { t } = this.props
-    return this.getTotal() > 1
-      ? t('escrowedPaymentReminderWithCount_plural', { count: this.getTotal() })
-      : t('escrowedPaymentReminder')
-  }
-  getCTA = () => {
-    return [
-      {
-        text: this.props.t('review'),
-        onPress: () => {
-          CeloAnalytics.track(CustomEventNames.escrowed_payment_review)
-          navigate(Stacks.EscrowStack)
-        },
-      },
-    ]
+  itemRenderer = (item: EscrowedPayment, key: number) => {
+    return <EscrowedPaymentLineItem payment={item} key={key} />
```

Comment:
Wondering, do we have some kind of ID we could use instead of the key?

Reply:
What would be the benefit to use ID vs key?

Reply:
I assumed the key here was an index in an array which should be avoided if possible:
https://reactjs.org/docs/lists-and-keys.html#keys